sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/NpmUpdateTasks.scala

```diff
@@ -21,12 +22,10 @@ object NpmUpdateTasks {
   def npmUpdate(baseDir: File,
```

Comment:
We should keep a deprecated overload of this method for compatibility reasons, which will delegate to this one.

Also the Scaladoc needs updating.